src/App.css

```diff
@@ -3,17 +3,90 @@
   justify-content: center;
   align-items: center;
   min-height: 2.5rem;
-  background-color: firebrick;
+  background-color: darkolivegreen;
   color: white;
   font-family: Georgia, "Times New Roman", Times, serif;
   font-weight: 600;
   letter-spacing: 1px;
 }
 
+.header-btn {
```

Comment:
Is it my imagination or is there a bunch of code that _looks_ like it's been changed in this PR, but was changed in a previous one? Maybe merging an up-to-date `master` branch into your branch will get rid of the extra code?

The main problem being that it's hard to tell what were the changes actually related to addressing this issue when there's other code showing up as green, too. Thanks!